test/runtests.py

```diff
@@ -94,7 +94,7 @@
 binary = args.binary
 if binary == None:
     if sys.platform == 'win32':
-        binary = 'Build/VcBuild/bin/{}_{}/ch.exe'.format(arch, flavor)
+        binary = 'Build\\VcBuild\\bin\\{}_{}\\ch.exe'.format(arch, flavor)
```

Comment:
>Build\\VcBuild\\bin\\{}_{}\\ch.exe [](start = 18, length = 34)

Just curious, is this change necessary? I've been running it on Windows and "/" worked.